db/silkworm/etl/collector_test.cpp

```diff
@@ -71,26 +71,16 @@ void run_collector_test(LoadFunc load_func) {
     // Load data
     auto to{txn->open(db::table::kHeaderNumbers)};
     collector.load(to.get(), load_func);
-
-    // Check wheter load was performed as intended
```

Comment:
Why remove this check?

Reply:
Instead of re-iterate over and over again now we do everything in load. so the check cannot be made.